Make build process never regenerate info.py

info.py is occasionally rebuilt in reproducible build tests, making the
package non reproducible.

The building is not needed since info.py is in the source tarball.
--- a/pavement.py
+++ b/pavement.py
@@ -149,7 +149,7 @@
 
 
 @task
-@needs("eyed3_info",
+@needs(# "eyed3_info",
        # "generate_setup",
        # "minilib",
        "setuptools.command.build")
@@ -226,7 +226,7 @@
 
 @task
 @needs("distclean",
-       "eyed3_info",
+       # "eyed3_info",
        # "generate_setup",
        # "minilib",
        "setuptools.command.sdist",
